<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_form</class>
 <widget class="QWidget" name="main_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ابزارهای مهندسین عمران</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/civil-engineering.jpg</normaloff>images/civil-engineering.jpg</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>21</y>
     <width>601</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>images/civil-engineering.png</normaloff>images/civil-engineering.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>350</width>
             <height>250</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;ابزارهای مهندسین عمران&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; dir='rtl' style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;مجموعه ابزارهای کدباز برای مهندسین عمران تحت لیسانس GPL v3&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://ebrahimraeyat.blog.ir/&quot;&gt;&lt;span style=&quot; color:#2980b9;&quot;&gt;وبلاگ توسعه دهنده نرم افزار&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#2980b9;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>10</number>
        </property>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>10</y>
           <width>196</width>
           <height>444</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="section_Button">
            <property name="text">
             <string>    مقاطع معادل</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>applications/section/images/2IPEPL.png</normaloff>applications/section/images/2IPEPL.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cfactor_Button">
            <property name="text">
             <string>    ضریب زلزله</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>applications/cfactor/images/cfactor.png</normaloff>applications/cfactor/images/cfactor.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="punch_Button">
            <property name="text">
             <string>    برش پانچ</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>applications/punch/images/main2.png</normaloff>applications/punch/images/main2.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="record_Button">
            <property name="text">
             <string>    رکورد زلزله</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>applications/records/images/Earthquake.png</normaloff>applications/records/images/Earthquake.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="exit_Button">
        <property name="text">
         <string>    خروج</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>images/quit.svg</normaloff>images/quit.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="update_Button">
        <property name="text">
         <string>    آپدیت</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>images/software-update.png</normaloff>images/software-update.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="about_Button">
        <property name="text">
         <string>   درباره</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>images/about.png</normaloff>images/about.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
